doc/news/changes/minor/20171017MartinKronbichler

```diff
@@ -0,0 +1,5 @@
+New: The class Utilities::MPI::Partitioner now supports to handle only a
```

Comment:
i always thought that verb `support` has to be used with gerund, i.e. `supports handing...`. This link suggests that it's indeed the case https://www.engvid.com/english-resource/verbs-followed-by-gerunds-and-infinitives/

Reply:
That is correct.